Classes/OldModel/Event.php

```diff
@@ -4571,4 +4551,31 @@ public function isPublished(): bool
     {
         return !$this->hasRecordPropertyString('publication_hash');
     }
+
+    private function getPaymentMethodsAsArray(): array
```

Comment:
Please let's move the shared code between `getPaymentMethodsAsArray` and `getPlacesAsArray` into a private method `getAssociatonAsArray(string $associatonFieldName, string $foreignTableName, string[] $foreignFields): array`.

Reply:
What is the benefit? How would this method be called from the others?

Reply:
`getPaymentMethodsAsArray` and `getPlacesAsArray` share the same structure and are only different in the data. So we reduce duplication with this.

`getPaymentMethodsAsArray` then could look like this:
```php
return $this->getAssociatonAsArray('tx_seminars_payment_method', 'tx_seminars_seminars_payment_methods_mm', ['title', 'description']);
```

I made a mistake with the signature above. It should be like this instead:
```php
getAssociatonAsArray(string $foreignTableName, string $mmTableName, string[] $foreignFields): array
```